scripts/ethereum/main.go

```diff
@@ -365,18 +363,18 @@ func redeem() []byte {
 		return nil
 	}
 
-	//acc := accReply.Accounts[0]
+	acc := accReply.Accounts[0]
```

Comment:
Are these changes related to this pr?

Reply:
Are these changes only related to integration/unit tests?